xwiki-platform-core/xwiki-platform-annotations/xwiki-platform-annotation-ui/src/main/resources/AnnotationCode/Translations.xml

```diff
@@ -113,8 +113,8 @@ AnnotationCode.AnnotationConfig_exceptionSpaces=Except for the following locatio
 AnnotationCode.AnnotationConfig_exceptionSpaces.hint=Deactivate annotations for the following subset of locations:
 AnnotationCode.AnnotationConfig_displayed=Display annotations by default
 AnnotationCode.AnnotationConfig_displayed.hint=Decide if annotations are displayed by default or only when "Show annotations" is checked on each page
-AnnotationCode.AnnotationConfig_displayHighlight=Display annotations highlight by default
-AnnotationCode.AnnotationConfig_displayHighlight.hint=Decide if the annotation's highlight should be displayed always or only when hovering over the annotation or the structure of annotations
+AnnotationCode.AnnotationConfig_displayHighlight=Display annotations' highlight by default
+AnnotationCode.AnnotationConfig_displayHighlight.hint=Decide if the annotations' highlight should be displayed always or only when hovering over the annotations or the structure of annotations
```

Comment:
Same as above. Focus on the action of "highlighting an annotation" and not on the "highlight of an annotation".

Also, why not `an annotation` and `a structure of annotations`? Sounds better then `the`, since it's a general setting and not something specific to a particular annotation.

Reply:
Done.